tests/foreman/destructive/test_installer.py

```diff
@@ -57,7 +57,7 @@ def test_installer_sat_pub_directory_accessibility(target_sat):
     https_curl_command = f'curl -i {target_sat.url}/pub/ -k'
     for command in [http_curl_command, https_curl_command]:
         accessibility_check = target_sat.execute(command)
-        assert 'HTTP/1.1 200 OK' in accessibility_check.stdout.split('\r\n')
+        assert 'HTTP/1.1 200 OK' or 'HTTP/2 200 ' in accessibility_check.stdout.split('\r\n')
```

Comment:
can you pls elaborate on why the command returns `HTTP/2 200`? non-blocking

Reply:
My guess is that `http_curl_command` returns the `HTTP/1.1` and `https_curl_command` returns the `HTTP/2`. Or the other way around? Either way we need both because each one is using a different protocol

Reply:
Correct. Most tools don't implement HTTP/2 over plain text HTTP (AKA `h2c`). None of the browsers do, not sure about curl. `http://` will in pretty much all of the cases use HTTP/1.1 while `https://` can upgrade to HTTP/2.

I think the reason for the space is just a format string. In HTTP/1(.1) it includes a human readable status code but in HTTP 2 I think they leave it out for efficiency. If you use a format string like `'%s %s %s' % (protocol, code, text)`  you will have an empty space when `text` is empty. Though on this part I'm just guessing.